packages/webpack-cli/lib/utils/cli-flags.js

```diff
@@ -144,6 +144,7 @@ const core = [
         alias: 't',
         type: String,
         group: ADVANCED_GROUP,
+        multiple: typeof cli !== 'undefined',
```

Comment:
```suggestion
        multiple: cli !== undefined,
```

Reply:
I did this for the sake of consistency.

https://github.com/webpack/webpack-cli/blob/4ee08d85d09e99f61d812519a70adb6047ff8581/packages/webpack-cli/lib/utils/cli-flags.js#L262

otherwise, I have no strong opinions.

Reply:
Let's change this too